docs/getting-started.md

```diff
@@ -194,9 +194,16 @@ API_SOCKET="/tmp/firecracker.socket"
 sudo rm -f $API_SOCKET
 
 # Run firecracker
-sudo ./firecracker --api-sock "${API_SOCKET}"
+sudo ./firecracker --api-sock "${API_SOCKET} --enable-pci"
 ```
 
+The `--enable-pci` flag instructs Firecracker to create all VirtIO devices using
```

Comment:
do we actually have to explain it here? Can we just add a link to the explanation in the kernel policy doc? My thinking is we shouldn't be inflating the getting started too much.

Reply:
My thinking was that it would be weird to add an optional flag there without explaining what it is, but I see your point. I don't have strong feelings, so if you feel it is bloating things a lot, I'm happy to remove from here.